src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonEntityMigrator.kt

```diff
@@ -1,12 +0,0 @@
-package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.prisonperson
-
-interface PrisonPersonEntityMigrator<NOMIS_RESPONSE, DPS_REQUEST> {
-  suspend fun migrateEntity(offenderNo: String): List<Long> =
-    getNomisEntity(offenderNo)
-      .toDpsMigrationRequests()
-      .migrate(offenderNo)
-
-  suspend fun getNomisEntity(offenderNo: String): NOMIS_RESPONSE
-  suspend fun NOMIS_RESPONSE.toDpsMigrationRequests(): List<DPS_REQUEST>
-  suspend fun List<DPS_REQUEST>.migrate(offenderNo: String): List<Long>
```

Comment:
Got rid of the generic entity migrator and replaced above with slimmed down interface that doesn't expose the internal types which are just details:
```
interface EntityMigrator {
  suspend fun migrate(offenderNo: String): DpsResponse
}
```